docs/parameters.yaml

```diff
@@ -600,13 +612,39 @@ type: string
 default: /
 components: ["cache"]
 ---
+name: Cache.RunLocation
+description: >-
+  A directory where temporary configurations will be stored for the xrootd daemon
+  started by the cache.
+
+  For non-root servers, if $XDG_RUNTIME_DIR is not set, a temporary directory will
+  be created (and removed on shutdown)
+type: filename
+root_default: /run/pelican/xrootd/cache
+default: $XDG_RUNTIME_DIR/pelican/cache
+components: ["cache"]
+---
+name: Cache.NamespacePrefix
+description: >-
+  The filepath prefix at which a cache is registered in the registry. Note that this value is not expected to be set by the user.
```

Comment:
We need to register this as a viper variable. Previously the cache prefix was saved to Origin.NamespacePrefix, but if serving both a cache and an origin in the same process, we need a separate value for the Cache or it would just overwrite the Origin's.